data/overview.yaml

```diff
@@ -233,7 +233,7 @@ Bilinear and quadratic forms over a vector space:
     Classification of O(2, R):
     Classification of O(3, R):
 
-5.
+5. 
```

Comment:
Any reason to add this space?

Reply:
No, garbage because I don't do some postprocessing after writing stuff ; though, note that all those numbers will disappear or become comments in #33

Reply:
Done